keps/sig-node/2411-cri-container-log-rotation/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: CRI Container Log Rotation
+kep-number: 2411
+authors:
+  - "@umohnani8"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-instrumentation
+  - sig-storage
+status: implemented
+creation-date: 2021-02-02
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+prr-approvers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: table
```

Comment:
this should say "stable" :)

Reply:
oops, fixed :)

Reply:
FYI -- I'm working on cleaning up the validation logic in https://github.com/kubernetes/enhancements/pull/2409.

ref: https://kubernetes.slack.com/archives/C1L57L91V/p1612288049074600, https://kubernetes.slack.com/archives/C1L57L91V/p1612385943085000